Source/WebCore/PAL/pal/spi/cocoa/CoreCryptoSPI.h

```diff
@@ -65,6 +65,17 @@ const struct ccdigest_info *ccsha512_di(void);
 extern const struct ccdigest_info ccsha512_ltc_di;
 
 void cccurve25519(ccec25519key out, const ccec25519secretkey sk, const ccec25519base);
+
+#if HAVE(UPDATED_CORE_CRYPTO_SIGNATURES)
+
+int cccurve25519_make_priv(struct ccrng_state *, ccec25519secretkey);
+
```

Comment:
nit: no need for separating all of these with spaces

Reply:
removed.